python/dgl/nn/pytorch/gt/spatial_encoder.py

```diff
@@ -87,51 +90,50 @@ class SpatialEncoder3d(nn.Module):
     `One Transformer Can Understand Both 2D & 3D Molecular Data
     <https://arxiv.org/pdf/2210.01765.pdf>`__
 
-    This module encodes pair-wise relation between atom pair :math:`(i,j)` in
+    This module encodes pair-wise relation between node pair :math:`(i,j)` in
     the 3D geometric space, according to the Gaussian Basis Kernel function:
 
-    :math:`\psi _{(i,j)} ^k = -\frac{1}{\sqrt{2\pi} \lvert \sigma^k \rvert}
+    :math:`\psi _{(i,j)} ^k = \frac{1}{\sqrt{2\pi} \lvert \sigma^k \rvert}
```

Comment:
Was `-` in the previous version a mistake?

Reply:
The paper has a negative sign but the implementation doesn't include it. They are actually equivalent in training because of linear projection after gaussian kernel computation.
We follow the formula in their implementation for pre-trained weights alignment.

Reply:
Got it, thanks.